main.go

```diff
@@ -122,28 +131,52 @@ func main() {
 
 	go Listen(port)
 	fmt.Println("Listening on port " + port + " - http://localhost:" + port + "/\n")
-	buildGraph()
+
+	// TODO: Workaround: this initial call is needed only to populate RateLmitSpecs
+	buildGraph(-1)
+
 	var duration time.Duration
 	if delay != (60 * pages) {
 		duration = time.Second * time.Duration(delay)
 	} else {
-		if reflect.TypeOf(RateLimitSpecs.PollInterval).String() != "int" {
```

Comment:
Why? IIRC this is needed to avoid panics if the first call status is 403 (rate limited), as with that status the `x-poll-interval` header is not included in the API's answer.